main/src/main/scala/org/clulab/numeric/mentions/package.scala

```diff
@@ -121,6 +135,34 @@ package object mentions {
       throw new RuntimeException(s"ERROR: cannot convert mention of type ${m.getClass.toString} to DateRangeMention!")
   }
 
+  /** Makes a specific date range when the year is combined with season, e.g., from August 23 to October 10 in 2017WS */
+  def toDateRangeMentionWithVagueSeason(mention: Mention): DateRangeMention =  mention match {
+    case m: DateRangeMention => m
+
+    case m: RelationMention =>
+      val date1Norm = getArgNorm("date1", m)
+      if(date1Norm.isEmpty)
+        throw new RuntimeException(s"ERROR: could not find argument date1 in mention [${m.raw.mkString(" ")}]!")
+      val date2Norm = getArgNorm("date2", m)
+      if(date2Norm.isEmpty)
+        throw new RuntimeException(s"ERROR: could not find argument date2 in mention [${m.raw.mkString(" ")}]!")
+
+      new DateRangeMention(
```

Comment:
I see quite a few instances of "new DateRangeMention(" with lots of similar code.  It might be time to add another constructor to DateRangeMention that takes the old mention and the two date#Norm Strings and calls the primary constructor.  Another option is to write a helper method that looks similar:
```scala
def newDateRangeMention(mention: Mention, date1Norm: String, date2Norm; String): DateRangeMention = {
  new DateRangeMention(
    mention.labels,...
```

Reply:
@kwalcock would you check if my last commit is what you meant with this comment?

Reply:
An apply method on the companion object is just as good or even better.  Thanks.